ui/src/app/components/webcomponents/action-step-form.component.ts

```diff
@@ -1850,10 +1852,13 @@ export class ActionStepFormComponent extends BaseComponent implements OnInit {
   }
 
   runtimeVariableAfterClose(data) {
-    if(data) {
+    if (data) {
       this.assignDataValue(this.getDataTypeString(TestDataType.runtime, data), true);//, this.testDataPlaceholder(), false);
-    } else  {
+    } else {
       this.showTestDataPopup();
     }
   }
+  public extractStringByDelimiterByPos(str:string){
```

Comment:
func name can be renamed since, we are not getting any pos or delimiter in the arguments